src/SubscribeMe/Subscriber/OxiMailingSubscriber.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SubscribeMe\Subscriber;
+
+use JsonException;
+use Psr\Http\Client\ClientExceptionInterface;
+use SubscribeMe\Exception\ApiResponseException;
+use SubscribeMe\Exception\CannotSendTransactionalEmailException;
+use SubscribeMe\Exception\CannotSubscribeException;
+use SubscribeMe\Exception\ApiCredentialsException;
+use SubscribeMe\Exception\UnsupportedTransactionalEmailPlatformException;
+use SubscribeMe\GDPR\UserConsent;
+use SubscribeMe\ValueObject\EmailAddress;
+
+class OxiMailingSubscriber extends AbstractSubscriber
+{
+    use ResponseValidationTrait;
+    public function getPlatform(): string
+    {
+        return 'oximailing';
+    }
+
+    /**
+     * @see https://api.oximailing.com/doc/#/contacts/post_lists__ListId__contacts
+     * @inheritdoc
+     */
+    public function subscribe(string $email, array $options, array $userConsents = []): bool|int
+    {
+        if (!is_string($this->getApiKey())) {
+            throw new ApiCredentialsException();
+        }
+
+        if (!is_string($this->getApiSecret())) {
+            throw new ApiCredentialsException();
+        }
+
+        $body = [
+            'mode' => $options['mode'],
+            'contacts' => [$email],
```

Comment:
Data structure does no allow options here.

You need to support additional data for an email.